apps/portal/app/components/follow/follow-form.tsx

```diff
@@ -32,7 +24,8 @@ import FollowReview from './follow-review'
 
 interface FollowFormProps {
```

Comment:
Perfect!
We'll finetune this as we fully iron out the actual type we'll end up using